libirods_rule_engine_plugin-elasticsearch.cpp

```diff
@@ -476,7 +476,12 @@ namespace {
 
             std::optional<nlohmann::json> jsonarray;
             get_metadata_for_object_index_id( _rei, object_id, is_coll, jsonarray );
-            obj_meta ["metadataEntries"] = *jsonarray; // dwm - check
+            if (!jsonarray) {
+                    irods::log( LOG_WARNING, fmt::format("Abort indexing metadata, null AVU array returned for object {} in {}:{}",
+                                               _object_path, __FILE__, __FUNCTION__));
```

Comment:
i think we're growing a convention of putting FILE, FUNCTION at the front of a log message... @alanking ?

Reply:
Changing per convention